samples/samples.mjs

```diff
@@ -7,16 +7,20 @@
 
 /** @type {Array<{title: string; file: string; shots: number}>} */
 export default [
-    {title: "Minimal", file: "./language/GettingStarted.qs", shots: 100},
-    {title: "Superposition", file: "./algorithms/Superposition.qs", shots: 100},
-    {title: "Entanglement", file: "./algorithms/Entanglement.qs", shots: 100},
-    {title: "Bell States", file: "./algorithms/BellState.qs", shots: 100},
-    {title: "Teleportation", file: "./algorithms/Teleportation.qs", shots: 1},
-    {title: "Random Bit", file: "./algorithms/RandomBit.qs", shots: 100},
-    {title: "Random Number Generator", file: "./algorithms/QRNG.qs", shots: 1000},
-    {title: "Deutsch-Jozsa", file: "./algorithms/DeutschJozsa.qs", shots: 1},
-    {title: "Bernstein–Vazirani", file: "./algorithms/BernsteinVazirani.qs", shots: 1},
-    {title: "Grover's search", file: "./algorithms/Grover.qs", shots: 100},
-    {title: "Hidden shift", file: "./algorithms/HiddenShift.qs", shots: 1},
-    {title: "Shor", file: "./algorithms/Shor.qs", shots: 1},
+    { title: "Minimal", file: "./language/GettingStarted.qs", shots: 100 },
+    { title: "Superposition", file: "./algorithms/Superposition.qs", shots: 100 },
+    { title: "Entanglement", file: "./algorithms/Entanglement.qs", shots: 100 },
+    { title: "Bell States", file: "./algorithms/BellState.qs", shots: 100 },
+    { title: "Teleportation", file: "./algorithms/Teleportation.qs", shots: 1 },
+    { title: "Random Bit", file: "./algorithms/RandomBit.qs", shots: 100 },
+    { title: "Random Number Generator", file: "./algorithms/QRNGNISQ.qs", shots: 1000 },
+    { title: "Random Number Generator (Advanced)", file: "./algorithms/QRNG.qs", shots: 1000 },
+    { title: "Deutsch-Jozsa", file: "./algorithms/DeutschJozsaNISQ.qs", shots: 1 },
+    { title: "Deutsch-Jozsa (Advanced)", file: "./algorithms/DeutschJozsa.qs", shots: 1 },
```

Comment:
Feels a bit strange to have "advanced" designation for a normal case and no designation for "dumbed down" case. Of course, this is all subjective, but to me "advanced" is a normal case. But I cannot come up with a better way to name these so it's up to you. You can keep it this way.

Reply:
Yes, it is a bit strange to have duplicates of the same samples and naming it is hard. I'll keep it this way for now.